src/docs/output-targets/dist.md

```diff
@@ -32,38 +32,37 @@ Luckily, both builds can be generated at the same time, and shipped in the same
+| `dir`                                     | The `dir` config specifies the public distribution directory. This directory is commonly the `dist` directory found within [npm packages](https://docs.npmjs.com/getting-started/packages). This directory is built and rebuilt directly from the source files. Additionally, since this is a build target, all files will be deleted and rebuilt after each build, so it's best to always copy source files into this directory. It's recommended that this directory not be committed to a repository.                                                                                                                                                                                                                           | `dist`  |
+| `empty`                                   | By default, before each build the `dir` directory will be emptied of all files. To prevent this directory from being emptied, change this value to `false`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        | `true`  |
+| `transformAliasedImportPathsInCollection` | This option will allow [path aliases](https://www.typescriptlang.org/docs/handbook/module-resolution.html#path-mapping) defined in a project's `tsconfig.json` to be transformed into relative paths in the output code for the implicit `dist-collection` output target (i.e. something like `import * as utils from '@my-utils-path';` will become `import * as utils from '../path/to/utils';`). This does not affect imports for external packages. Currently, aliased paths are not transformed in the emitted code after running through the TS compiler.<br><br>If using the `dist-collection` output target directly, the same result can be achieved using the `transformAliasedImportPaths` flag on the target's config. | `false` |
```

Comment:
The description for `transformAliasedImportPathsInCollection` is probably a little _too_ much information for a table cell. In it, we describe:
- path aliases in TS
- mention the implicit `dist-collection` output target, which is opaque to most users (they don't know that it's implicitly added, which may be confusing)
- provide an example of the transform
- state what this doesn't affect
- describe what it means to use `dist-collection` directly

I would consider breaking some of this out into a `Config` sections, like https://stenciljs.com/docs/custom-elements#config.

Reply:
This is what I was saying yesterday with some targets having different ways of documenting their configs. I went ahead and moved everything for `dist` out of a table format and into the structure used for `dist-custom-elements`. Definitely feels a bit cleaner